app/views/layouts/application.html.erb

```diff
@@ -239,6 +239,7 @@
               <% if request.url.include?('/docs/integrations') %>
                 <p class="Docs__nav__section-heading">Integrations</p>
                 <ul class="Docs__nav__sub-nav">
+                  <li class="Docs__nav__sub-nav__item"><%= sidebar_link_to "GitHub", 'integrations/github' %></li>
```

Comment:
I think we need to do some extra thing (maybe a route or something?) somewhere to make this one the auto-linked page from Integrations. At the moment it's doing this when you go to integrations 😊 

![image](https://user-images.githubusercontent.com/2074469/77129523-02a68300-6aa0-11ea-8ea6-92c6b24da0eb.png)